baus/models.py

```diff
@@ -509,6 +509,12 @@ def add_extra_columns_func(df):
               df.deed_restricted_units.sum())
     df["preserved_units"] = 0.0
 
+    if "inclusionary_units" not in df.columns:
```

Comment:
Going to ask some question to help me understand these models- is this because the developer may add inclusionary itself, but if it doesn't we need to add it here

Reply:
Yes, `residential_developer()` and `subsidized_residential_developer()` may add inclusionary, but `retail_developer()`, `office_developer()`, and `scheduled_development()` don't. So if we don't add this to `add_extra_columns_func()`, there will be an 'inclusionary_units' column doesn't exist error.